plugins/win-capture/game-capture.c

```diff
@@ -537,7 +565,8 @@ static void *game_capture_create(obs_data_t *settings, obs_source_t *source)
 
 	gc->source = source;
 	gc->initial_config = true;
-	gc->retry_interval = DEFAULT_RETRY_INTERVAL;
+	gc->retry_interval = DEFAULT_RETRY_INTERVAL*
```

Comment:
Style:  Typically, you need to surround math operators by a single space on each side.  Don't leave a trailing space at the end of the line either, so in this case:
```
	gc->retry_interval = DEFAULT_RETRY_INTERVAL *
```